README.md

```diff
@@ -28,7 +28,6 @@ You can install the extension from one of the following links
 Please ensure that you have at least the **minimum** recommended versions
 
 -   Node - 10.15.0 (Check by running `node --version`) - This is the version being enforced on our builds
--   NPM - 6.4.1 (Check by running `npm --version`)
```

Comment:
Users need a global install of yarn, no? Shouldn't we note that here?

Reply:
Yes. Let me make a note on that. User do need `yarn` for this to work.